_episodes/04-recipe.md

```diff
@@ -222,7 +223,7 @@ Please note the following sections:
 {: .solution}
 
 > ## What do you think running this recipe will produce?
-> A time series plot of thetaoga with increements of 1 year.
+> A time series plot of thetaoga with increments of 1 year.
 {: .solution}
 
 > ## Not all parts of the recipe are mandatory
```

Comment:
I think it would be nice to list here the parts of the recipe that are and are not mandatory, not only provide the example about the documentation.